Entities/Objects/Weapons/Guns/Ammunition/Projectiles/pistol.yml

```diff
@@ -1,17 +1,17 @@
 - type: entity
   id: BulletPistol
-  name: bullet (.35 auto)
+  name: bullet (9x19mm PSTF)
   parent: BaseBullet
   categories: [ HideSpawnMenu ]
   components:
   - type: Projectile
     damage:
       types:
-        Piercing: 21
+        Piercing: 17
```

Comment:
this makes it have retardingly sloppy ttk since vs 50% armor value it means it would do 8 dmg on average